.env-dist

```diff
@@ -145,3 +145,7 @@ STATS_TOKEN=
 
 # Support article link
 NEXT_PUBLIC_MONITOR_SUPPORT_URL=https://support.mozilla.org/kb/firefox-monitor
+
+# GCP PubSub Project ID and subscription name
+GCP_PUBSUB_PROJECT_ID=
+GCP_PUBSUB_SUBSCRIPTION_NAME=hibp-breaches-sub
```

Comment:
From some random code below, but do we need to add `GCP_PUBSUB_TOPIC_NAME` to the .env-dist file too?
```js
const projectId = process.env.GCP_PUBSUB_PROJECT_ID;
const topicName = process.env.GCP_PUBSUB_TOPIC_NAME;
const subscriptionName = process.env.GCP_PUBSUB_SUBSCRIPTION_NAME;
```

Reply:
Yes, I noticed that after landing, will file a small followup. The code detects if any of these aren't set and error out with a specific message, but I think having it in the `.env-dist` is still the right thing to do.

Reply:
https://github.com/mozilla/blurts-server/pull/3387/